### ISSUE TITLE
Boundary lines in DetailsList dont match Color contrast standards

### ISSUE THREAD

**Issue by nkalpana** [OfficeDev/office-ui-fabric-react#11485]
### Environment Information

- **Package version(s)**: Latest version (one used by Controls Playground when we do export a sample to Codepen)
- **Browser and OS versions**: Windows 10, Chrome/Edge

### Describe the issue:
The color contrast of the boundary lines of DetailsList, don't pass the color contrast standard ratio (> 3:1 or > 4.5: 1)

How to repro:
1) Open code pen from below
2) Download Accessibility insights for Windows, desktop app.
3) Go to Color contrast tool (palette icon on the extreme left hand side menu)
4) Click the icon for the Color one, and then use the magnifier to click on the color of any boundary line
5) Click the icon for Color two, and then use the magnifier to click on the color of the background.

The ratio will be displayed as 1.8 : 1, and it will indicate that as a failure because its lesser than 3 : 1.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/nkalpana/pen/LYEbzMO

#### Actual behavior:

The current ratio is 1.189 : 1

#### Expected behavior:

Should be greater than 3: 1

**Comment by nkalpana** [comment]
![DetailsListColorContrastBoundaryLines](https://user-images.githubusercontent.com/7914826/70948028-6c317400-200f-11ea-8ed4-39b1855e2d29.gif)


(a GIF video showing how you can use the Accessibility Insights tool to check the color contrast)

**Comment by ecraig12345** [comment]
I'm not sure whether the row borders are subject to color contrast requirements, since it's still possible to tell which items go together based on their horizontal alignment (the tool does sometimes flag things when it shouldn't). Do you have a reference for a specific part of the ARIA standard which requires *specifically* row borders to meet 3:1 contrast?
